listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -264,8 +265,13 @@ export default class ListenCard extends React.Component<
                       <span
                         className="listen-time text-muted"
                         title={
-                          listen.listened_at_iso?.toString() ||
-                          new Date(listen.listened_at * 1000).toISOString()
+                          listen.listened_at
```

Comment:
there are 2 `<MediaQuery>` elements in this component, so it should be updated in both